radicle-cli/src/commands/issue.rs

```diff
@@ -232,13 +257,18 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
                 )?;
             }
         }
-        Operation::List => {
+        Operation::List { assignee } => {
             let mut t = term::Table::new(term::table::TableOptions::default());
             for result in issues.all()? {
                 let (id, issue, _) = result?;
+                let assigned: Vec<_> = issue.assigned().collect();
+
+                if Some(true) == assignee.map(|a| !assigned.contains(&&a)) {
+                    continue;
+                }
```

Comment:
We don't need to call `issuee.assigned()` if `assignee` is `None`, so I'd rewrite this with that in mind.

Reply:
Hmm its used a little further below to be displayed to the users.  Or did I miss something?

The assignee is intended to be whom to list issues for.  The None part of the option is to specify an 'all'.  Its an optional argument.